modules/oc-adm-by-example-content.adoc

```diff
@@ -82,6 +82,17 @@ Deny a certificate signing request
 == oc adm copy-to-node
 Copies specified files to the node.
 
+.Example usage
+[source,bash,options="nowrap"]
+----
+  # copy a new bootstrap kubeconfig file to every node
+  oc adm copy-to-node --copy=new-bootstrap-kubeconfig=/etc/kubernetes/kubeconfig
```

Comment:
The command will failed with error :
`oc adm copy-to-node --copy=new-bootstrap-kubeconfig=/etc/kubernetes/kubeconfig-test
error: unable to get interesting nodes: You must provide one or more resources by argument or filename.
Example resource specifications include:
   '-f rsrc.yaml'
   '--filename=rsrc.json'
   '<resource> <name>'
   '<resource>'`